platform/9.1/_index.md

```diff
@@ -52,20 +52,20 @@ The ballot was run in the [jakarta.ee-spec mailing list](https://www.eclipse.org
 
 ## Release Review 
 
-The Specification Committee Ballot concluded successfully on 2021-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2021-05-21 with the following results.
```

Comment:
Didn't the vote complete on Monday, May 17? Or, the 18th at the latest.  The 21st is in the future...  :-)